src/area/uploadArea.ts

```diff
@@ -70,6 +70,7 @@ class UploadArea {
         (options.allowDragDrop === undefined || options.allowDragDrop === null ? true : options.allowDragDrop);
         this.options.clickable = options.clickable === undefined || options.clickable === null ? true : options.clickable;
         this.options.accept = options.accept || '*.*';
+        this.options.validateExtension = options.validateExtension === undefined || options.validateExtension === null ? false : options.validateExtension;
```

Comment:
The defualt value is false so 'this.options.validateExtension = !!options.validateExtension' should be enough.